src/library/tests/test_views.py

```diff
@@ -145,3 +146,22 @@ def test_import_library_to_current_deck(self):
 
         # Ensure that the newly imported quest is not visible to students
         self.assertFalse(quest_qs.get().visible_to_students)
+
+    def test_side_bar_library_drop_down(self):
+        """ Checks if library drop down is available when siteconfig.enable_shared_library is true """
+        staff = baker.make(User, is_staff=True)
+        self.client.force_login(staff)
+
+        config = SiteConfig.get()
+
+        # if `enable_shared_library=False` then "Quest Library" should not exist
+        config.enable_shared_library = False
+        config.save()
+        response = self.assert200('library:library_quest_list')
+        self.assertNotContains(response, 'Quest Library')
+
+        # if `enable_shared_library=True` then "Quest Library" should exist
+        config.enable_shared_library = True
+        config.save()
+        response = self.assert200('library:library_quest_list')
+        self.assertContains(response, 'Quest Library')
```

Comment:
Since this is checking anywhere on the page, and that text COULD appear elsewhere, check more specifically for the button with something like `</i>&nbsp; Quest Library</a>`

Also you can remove the Flip horizontal class, I think that's only on the quest one to make the icon look a bit different from the other Quest /shield icon.

Reply:
done